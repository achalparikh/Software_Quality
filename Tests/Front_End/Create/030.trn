10_admin_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _