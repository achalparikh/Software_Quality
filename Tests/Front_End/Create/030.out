Account holder name too long, could not create account.